pkg/allowrules/allowrules.go

```diff
@@ -0,0 +1,73 @@
+package allowrules
+
+import (
+	"fmt"
+)
+
+type TransportProtoAllowRule struct {
+	serviceName string
+	port        uint16
+}
+
+var tcpAllowRules = []AllowRules{
+	TransportProtoAllowRule{
+		"Kubernetes API",
+		6443,
+	},
+	TransportProtoAllowRule{
+		"ETCD",
+		2380,
+	},
+	TransportProtoAllowRule{
+		"ETCD",
+		2379,
+	},
+	TransportProtoAllowRule{
+		"SSH",
+		22,
+	},
+}
+
+var udpAllowRules = []AllowRules{
+	TransportProtoAllowRule{
+		"DHCP",
+		68,
+	},
+}
+
+func GetTCP() []AllowRules {
+	return tcpAllowRules
+}
+
+func GetUDP() []AllowRules {
+	return udpAllowRules
+}
+
+type AllowRules interface {
+	//IsConflictTransport checks to see if a range or a single port conflicts with the allow rule. If endPort is 0, we assume
+	// only a single port and not a range.
+	IsConflictTransport(startPort, endPort uint16) (bool, error)
```

Comment:
I don't think this belong to this pkg
also can u add method to get protocol and the port

Reply:
Yeah, removed the interface. Pointless.
No need to add get protocol because you can pull the fail safes by protocol so you know what protocol you are getting.